modules/ROOT/pages/Development/BeginnersGuide/Adding_Ingame_Mod_Icon.adoc

```diff
@@ -12,13 +12,28 @@ there's still a couple steps needed for Alpakit to recognize it
 and package it with your mod.
 
 
-== Setup
+== Select an Image
 
 First, we need a png file to use as the Icon for our mod.
 You can re-use the same one you plan to use when
 xref:UploadToSMR.adoc[uploading your mod to SMR],
 but you might need to resize it for use here - this icon should be a maximum of 128x128 pixels. 
 
+== Alpakit Helper Utility
+
+If you created your mod using the
+xref:Development/BeginnersGuide/SimpleMod/gameworldmodule.adoc[Alpakit Create Mod Utility],
+the relevant configuration files mentioned below have already been created for you.
+
+Simply replace the existing `Icon128.png` file located at `/Plugins/<mod_reference>/Resources`
+with your own icon image and proceed to 
+xref:#_making_sure_it_worked[make sure it worked].
```

Comment:
Regardless of whether the mod was made with Alpakit or not, the Edit Mod window can set the mod icon, as mentioned on the gameworldmodule page too